package.json

```diff
@@ -48,6 +48,17 @@
   "dependencies": {
     "body-parser": "^1.15.2",
     "d3": "^4.2.2",
+    "d3-array": "^1.0.1",
```

Comment:
Agree. Please remove all those dependencies and use d3 modules already installed in the project. If you need any help with this, let us know.